cilium/pre/values.yaml

```
@@ -20,6 +20,9 @@ enableIPv4Masquerade: false
 enableIdentityMark: false
 externalIPs:
   enabled: true
+extraArgs:
+  - "--endpoint-gc-interval"
+  - "0"
```

I suspect it would be easier to read?
```suggestion
  - "--endpoint-gc-interval=0"
```